Main/connect_mqtt.ino

```diff
@@ -1,20 +1,20 @@
-void connectMQTT()
```

Comment:
why this file is changed? i think you need to uncommit everything what has no actual changes. I guess it is just formatting (

Reply:
yeah this is bizarre, I only changed the delay interval temporarily but it says entire file was rewritten? :o lemme see what I can do